mobile/Ihc/screens/LoginScreen.js

```diff
@@ -72,22 +73,64 @@ class LoginScreen extends Component<{}> {
 
   render() {
     return (
-      <Container>
-        <Text style={styles.welcome}>
-          Welcome to clinic!
-        </Text>
+      <View style={{
+        flex: 1,
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+      }}>
+
+        <View style={{
+          height: '30%',
+          width: '100%',
+          }}>
+
+          <Image source={require('../images/ihc.png')} />
+
+        </View>
+
+        <View style={{
+          height: '70%',
+          width: '100%',
+          backgroundColor: '#0660AE',
+        }}>
 
-        <View style={styles.form}>
-          <Form ref="form"
-            type={this.credentials}
-            options={this.options}/>
         </View>
 
-        <Button onPress={this.submit}
-          text="Login"
-          style={styles.button}
-        />
+        <View style={{
+          width: 500,
+          backgroundColor: 'white',
+          position: 'absolute',
+          alignItems: 'center',
+          justifyContent: 'center',
+          padding: 50,
+          paddingLeft: 15,
+          paddingRight: 15,
+          top: 100,
+          shadowColor: "#000",
+          shadowOffset: {
+          	width: 0,
+          	height: 2,
+          },
+          shadowOpacity: 0.23,
+          shadowRadius: 2.62,
+          elevation: 4,
+        }}>
+
+        <Container>
```

Comment:
Please fix the indentation. <Container> should line up with </Container>, and all the children should be indented one block in. Otherwise it's hard to read which elements are grouped together